routers/web/repo/issue.go

```diff
@@ -291,6 +291,8 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		return
 	}
 
+	ctx.Data["Posters"] = ctx.Data["Assignees"]
```

Comment:
I don't think this is right; I believe the list of users that can open a PR is not the same as the list of possible assignees, but I can't find an existing method to get the list of users that can open a PR. Is this available? If not, does anyone have pointers on how I could write this method, similar to `GetRepoAssignees()`?